documentation/user_docs/docs/manual/Cutting_data_of_interest_from_SQW_files_and_objects.rst

```diff
@@ -649,21 +649,21 @@ where:
    :math:`[0,1,0]` respectively.
 
 - ``type``  Three character string denoting the the projection normalization of each
-  dimension, one character for each directions, e.g. ``'add'``, ``'arr'``, ``'adr'``.
+  dimension, one character for each directions, e.g. ``'add'``, ``'hrr'``, ``'adr'``.
 
-  At the moment there is only one possible option for the first (length) component of ``type``:
+  There are 6 possible options defining scale for the value of the momentum transfer:
 
-  1. ``'a'``     Inverse angstroms.
-
-  ..
-     2. ``'r'``
-
-        Reciprocal lattice units (r.l.u.) which normalises so that the maximum of
-        :math:`|h|`, :math:`|k|` and :math:`|l|` is unity.
-
-     3. ``'p'``
+  1. ``'a'``  :math:`|Q|` is measured in inverse angstroms.
+  
+  2. ``'r'``  Reciprocal lattice units (r.l.u.) which normalises vector :math:`\vec{Q}` so that the scale is the maximal value of the :math:`\vec{u}` projection to the unit vectors directed along the reciprocal lattice vectors.
 
-        Preserve the values of ``u`` and ``v``
+  3. ``'p'``   The scale is the length of the vector :math:`\vec{u}`
+  
+  4. ``'h'``   The scale is the length of the first vector of reciprocal lattice :math:`a^{*}`
+  
+  5. ``'k'``   The scale is the length of the second vector of reciprocal lattice :math:`b^{*}`
+  
+  6. ``'l'``   The scale is the length of the third vector of reciprocal lattice :math:`c^{*}`
```

Comment:
```suggestion
  4. ``'h'``   The scale is the length of the first vector of reciprocal lattice :math:`a^{*}`
  
  5. ``'k'``   The scale is the length of the second vector of reciprocal lattice :math:`b^{*}`
  
  6. ``'l'``   The scale is the length of the third vector of reciprocal lattice :math:`c^{*}`
```
```suggestion
  4. ``'h'``   The scale is the length of :math:`a^{*}` the first vector of reciprocal lattice 
  
  5. ``'k'``   The scale is the length of :math:`b^{*}` the second vector of reciprocal lattice 
  
  6. ``'l'``   The scale is the length of :math:`c^{*}` the third vector of reciprocal lattice 
```
without this, each line gives the impression that the vector **is** the reciprocal lattice. i know that doesn't make sense, but it is what the phrasing would imply

Reply:
should it be dash here?

Reply:
or comma

Reply:
would not agree with this -- the scale is -- how it would be interpreted as the vector?

Reply:
but if you add dash or comma, I do not see the difference between mine and suggested forms -- would accept the change

Reply:
agree with comma. If you look at the original wording, it's "third vector of reciprocal lattice :math:`c^{*}`" and math:c is next to reciprocal lattice. math:c needs to be next to third vector, which is what I've done

Reply:
yes, I;ve commited the suggested change + comma